Makefile

```diff
@@ -86,7 +86,7 @@ check: manifests generate add-license lint test
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 .PHONY: test
 test: manifests generate fmt envtest check-license ## Run tests. Some test depend on Linux OS
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go run github.com/onsi/ginkgo/v2/ginkgo --label-filter="!integration" -coverprofile cover.out ./...
```

Comment:
we can install ginkgo as additional tool (similar way as controller-gen, kustomize, ...)

Reply:
We could. But we are installing tools into the `bin` folder for the sole reason to not having those dependencies in our `go.mod`. As `ginkgo` is need for testing and is hence a dependency this approach is valid.

Reply:
ok. i agree it isn't needed